.pre-commit-config.yaml

```diff
@@ -4,16 +4,16 @@ repos:
     hooks:
       - id: system
         name: mypy
-        entry: poetry run tox -e mypy
+        entry: poetry run tox -e mypy-locked
+        pass_filenames: false
+        language: system
+  - repo: local
+    hooks:
+      - id: system
+        name: isort
+        entry: poetry run isort cyclonedx tests
         pass_filenames: false
         language: system
-#  - repo: local
```

Comment:
could you explain why `isort` was removed?

Reply:
Weirdly, this was put back in [7d8a1f1](https://github.com/CycloneDX/cyclonedx-python-lib/pull/263/commits/7d8a1f151c5ee08e8547ec533cd23a5cd10e452c) and is on this branch - so no idea why GH is calling this out still?